### PR TITLE
Fixing docs

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Cilium is...
 
 ## Networking Model
 
+<p align="center">
+   <img src="doc/cilium-world.png" width="500">
+</p>
+
```
Comment (tgraf) [inline README.md:46]: I think we should move this to the top.

**Comment by tgraf** [comment]
Looks great otherwise

File: README.md (hunk 1 above)
Comment (aanm) [inline README.md:46]: Between `Cilium provides fast and low latency in-kernel networking connectivity and security policy enforcement for containers based on eBPF.` and `Cilium is...`? or right after the Title?

Comment (tgraf) [inline README.md:46]: Right after the initial abstract

**Pull request merged**